test/coverage-golden/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, recurseIntoAttrs, runCommand, testSrc, compiler-nix-name, git, hpc-coveralls, jq }:
+
+with stdenv.lib;
+
+let
+  exampleProjectSrc = fetchFromGitHub {
+    owner  = "input-output-hk";
+    repo   = "cardano-shell";
+    rev    = "8e0d6f5548c1c13a9edfeb56e4ea27c0a6d10948";
+    sha256 = "07yjl3agiacmy5irxxmrmm7ffpnvz84zgrylnp3wmcrn417pc5fq";
+  };
```

Comment:
We could use niv here to make this easier to update.  Run `niv add input-output-hk/cardano-shell` in the root of haskell.nix and then use `exampleProjectSrc = (import ../../nix/source.nix {}).cardano-shell;` here.